.eslintrc

```diff
@@ -17,13 +19,15 @@
     "no-prototype-builtins": 0,
     "no-duplicate-imports": 0,
     "flowtype/require-valid-file-annotation": [2, "always", {"annotationStyle": "line"}],
-    "flowtype/semi": [2, "never"],
     "flowtype/boolean-style": [2, "boolean"],
     "flowtype/no-dupe-keys": 2,
-    "flowtype/space-before-type-colon": [2, "never"],
-    "flowtype/space-after-type-colon": [2, "always"],
     "flowtype/union-intersection-spacing": [2, "always"],
-    "flowtype/object-type-delimiter": 2
+    "flowtype/object-type-delimiter": 2,
+     "prettier/prettier": ["error", {
+      "semi": false,
```

Comment:
Can you move these settings into `.prettierrc` so that editors will see these settings too?

It is best to put all settings in there, including the default values, so that people's personal Prettier configuration doesn't interfere…